modules/tinymce/CHANGELOG.md

```diff
@@ -17,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Clicking beside a non-editable element could cause the editor to incorrectly scroll to the top of the document #TINY-7062
 - Split toolbar buttons incorrectly had nested `tabindex="-1"` attributes #TINY-7879
 - Fixed an exception getting thrown when there were more `col` elements than columns in a table #TINY-7041
+- Fixed an exception getting thrown when there were less `col` elements than columns in a table #TINY-8011
```

Comment:
```suggestion
- Fixed an exception getting thrown when there were fewer `col` elements than columns in a table #TINY-8011
```

Reply:
Should we just combine the 2 changelog entries here together? As having 2 separate ones doesn't really give any advantage and are likely to cause more confusion if anything. e.g
```
- Fixed an exception getting thrown when the number of `col` elements didn't match the columns in a table #TINY-7041 #TINY-8011
```

Reply:
Have combined the two changelog entries